Server/VikingVault.API/Startup.cs

```diff
@@ -27,7 +33,35 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDbContext<VikingVaultDbContext>
                 (options => options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
 
-            services.AddScoped<IBankService, BankService>();
+            // configure strongly typed settings objects
+            var appSettingsSection = Configuration.GetSection("AppSettings");
+            services.Configure<AppSettings>(appSettingsSection);
+
+            // configure jwt authentication
```

Comment:
Please remove this comment and create a private method with a suggestive name instead.